src/components/Tutorial/TutorialScrollUpButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC, useCallback } from 'react'
+import TutorialNavigationButton from './TutorialNavigationButton'
+
+interface TutorialScrollUpButtonProps {
+  show: boolean
+}
+
+/** A button to scroll up to the tutorial. */
+const TutorialScrollUpButton: FC<TutorialScrollUpButtonProps> = ({ show }) => {
+  /**
+   * Scrolls to the top of the window.
+   */
+  const scrollUp = useCallback(() => {
+    window.scrollTo({
```

Comment:
Can you use https://github.com/cybersemics/em/blob/a33035e2076d82d8114459d662b0508ca353d686/src/device/scrollTo.ts? It's the same thing, but we abstract direct access to `window` to decouple the application logic from the browser API. This makes it easier to patch browser incompatibilities and mock functionality in tests.

Reply:
Definitely! I didn't see this at first.

Refactored here: 6019ce443289fb1f59919af916d833c0abf8d967